sdk/servicebus/service-bus/src/serviceBusAtomManagementClient.ts

```diff
@@ -1788,7 +1787,7 @@ export class ServiceBusAdministrationClient extends ServiceClient {
    * @param ruleFilter Defines the filter expression that the rule evaluates.
    * @param operationOptions The options that can be used to abort, trace and control other configurations on the HTTP request.
    *
-   * Following are errors that can be expected from this operation
+   * Following are error that can be expected from this operation
```

Comment:
```suggestion
   * Errors that can be thrown from this operation
```

Maybe? I'm pretty sure either way that 'errors' was right, and 'error' is not correct.

Reply:
Prettier indents the whole code and messes up the file whenever I update JS Doc comments.
This was a result of that event. I'll retain the older comment.